app/services/idv/steps/inherited_proofing/verify_wait_step_show.rb

```diff
@@ -0,0 +1,34 @@
+module Idv
+  module Steps
+    module InheritedProofing
+      class VerifyWaitStepShow < VerifyBaseStep
```

Comment:
`VerifyBaseStep` inherits from `DocAuthBaseStep`, and may carry a _lot_ of baggage related to doc auth we may not interested in? I assume you've combed through these? I wonder how much of all that we're actually using/need for IP. Perhaps something for later, but if what is _common to both flows_ is not that substantial, perhaps it can be refactored out (into a module for example) so we can mix in only what IP needs.

Reply:
Definitely a good place for a refactor in the future! The "move along to the next page" part of the flow was stuck in here and it was kind of a pain to try to tease out, so I decided to just go with the baggage for now.

Reply:
Yea, there seems to be a bit of fat in there we don't need, but perhaps we can have a look at it once we get a breather.